lib/charms/hpc_libs/v0/slurm_ops.py

```diff
@@ -138,7 +146,7 @@ def _call(cmd: str, *args: str, stdin: Optional[str] = None) -> str:
     except subprocess.CalledProcessError as e:
         _logger.error(f"`{' '.join(cmd)}` failed")
         _logger.error(f"stderr: {e.stderr.decode()}")
-        raise
+        raise SlurmOpsError(f"Command {cmd[0]} failed. Reason:\n{e.stderr.decode()}")
```

Comment:
[nit]: should all error messages be lowercase or should they be titlelized?

At least in Go, the strong preference is that all error messages should be lowercase. What are your thoughts on this :eyes:?

Reply:
lowercase seems better for nice displays: `Error: important message` vs `Error: Important message`